pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -805,8 +805,8 @@ func getPidsForProcess(name, pidFile string) ([]int, error) {
 	}
 
 	// Try to lookup pid by process name
-	pids, err2 := procfs.PidOf(name)
-	if err2 == nil {
+	pids, pidsErr := procfs.PidOf(name)
+	if pidsErr == nil {
```

Comment:
seems like we could change this block and the one above to be like the following:

```
if pids, err := foo(name); err == nil {
 return pids
}
```

otherwise err and err2 in this same function are equally descriptive...

Reply:
@derekwaynecarr  the modification is completed, please review again.